_includes/specials/bech32/18-real-fees.md

```diff
@@ -0,0 +1,50 @@
+As of this writing, the price of bitcoin in national currencies has been
+rapidly increasing over the past few months.  We don't know whether this
+trend will continue (and it may reverse), but significant increases in
+the bitcoin price are notable in the context of bech32 because
+transaction fees are paid in bitcoin terms rather than dollar terms.
+That means the real cost to send a transaction is expected to increase
+in correspondence with bitcoin prices even if feerates stay the same.
+
+We've previously discussed [how much users and services can save][bech32 savings
+segment] by switching to native segwit (bech32) addresses, but we've
+only described that in terms of vbyte and percentage savings.  In this
+bech32 sending support section, we look at the savings in real terms.
+
+The lowest practical fee is 0.00000001 BTC/vbyte.  The highest typical fees
+seen to date were around 0.00001000 BTC/vbyte in December 2017 and January 2018.
+For that range, the following charts show the amount of money that may be saved for
+users of two common transaction templates, single-sig and 2-of-3
+multisig:
+
+![Single-sig legacy P2PKH versus segwit P2WPKH](/img/posts/2019-07-real-cost-p2pkh-p2wpkh.png)
+
+![2-of-3 multisig legacy P2SH versus segwit P2WSH](/img/posts/2019-07-real-cost-p2sh-p2wsh.png)
+
+For users of legacy transactions with other transaction templates,
+you may be able to learn roughly how much you will save in percentage
+terms by pasting in the txid of a typical transaction you sent into
+information sites such as an Esplora
+block explorer like [Blockstream.info][].  You can multiply that
+percentage by the vbyte size of your transaction to see how many vbytes
+you'd save.  Note that using a third-party service reveals to them that
+you have an interest in that transaction, possibly significantly
+reducing your privacy.  You can privately get the approximate vbyte
+savings by aggregating the byte size of all scriptSigs in your typical
+transaction and then multiplying that number by 0.75.  When you know how
```

Comment:
I think everything from "You can privately..." onward could be made clearer by turning it into step-by-step instructions, ie:

1. Use a tool to parse the transaction. Bitcoin Core comes bundled with a tool called `bitcoin-tx` that can do this for you. Run `bitcoin-tx -json <hex_serialized_tx>`
2. Add up the total size of all scriptSigs. `bitcoin-tx` outputs the hex of each scriptSig. Halve the length of this hex string to get the size of the scriptSig.
3. Multiply the size of scriptSigs that value by 0.75 to get vbytes saved.
4. Multiply the vbytes saved by feerate to get the amount of bitcoin saved.
5. Multiply the amount of bitcoin saved by the exchange rate to get savings.

Reply:
I like the step by step instructions. (note again, for a future newsletter)